WPMedia/fusion-news-theme-blocks
# THEMES-837: Category Carousel Block resizer update

## Description

This PR updates the category carousel block to use resizer v2.

## Jira Ticket

- [THEMES-837](https://arcpublishing.atlassian.net/browse/THEMES-837)

## Acceptance Criteria

- Category Carousel - Arc Block is using Resizer V2

## Test Steps

1. Checkout this branch `git checkout THEMES-837`
2. Checkout the `commerce` branch of the feature pack repo.
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/category-carousel-block`
4. Create a new page and add the 'Category Carousel' block to it.
5. Begin filling out the fields for the categories.
  - Once the label and link url are filled out, the category box should show in the pagebuilder preview area.
  - You can then click the 'Replace image' button to select an image and get the extra data needed to use resizer v2.
6. Once there are some categories fully filled out, ensure the images are using resizer v2 URLs.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-837]: https://arcpublishing.atlassian.net/browse/THEMES-837?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[THEMES-837]: https://arcpublishing.atlassian.net/browse/THEMES-837?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[THEMES-837]: https://arcpublishing.atlassian.net/browse/THEMES-837?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
